includes/wikia/services/ConsulService.class.php

```diff
@@ -0,0 +1,134 @@
+<?php
+class ConsulService extends Service {
+
+	const DATA_CENTER_SJC = "sjc";
+	const DATA_CENTER_POZ = "poz";
+	const DATA_CENTER_RES = "res";
+
+	const TAG_PRODUCTION = "production";
+	const TAG_TESTING = "testing";
+
+	const SERVICE_NAME_SCHEMA = "{tag}.{service}.service.{dataCenter}.consul";
+
+	protected $dataCenter = self::DATA_CENTER_SJC;
+
+	protected $serviceTag;
+	protected $serviceName;
+
+	public function __construct( $serviceName, $serviceTag=null ) {
+
+		$this->serviceName = $serviceName;
+
+		if ( $serviceTag === null ) {
+			$serviceTag = $this->isDevelopmentEnv() ? self::TAG_TESTING : self::TAG_PRODUCTION;
+		}
+
+		$this->serviceTag = $serviceTag;
+		return $this;
+	}
+
+	/**
+	 * @return bool
+	 */
+	protected function isDevelopmentEnv() {
+		return F::app()->wg->DevelEnvironment === true;
+	}
+
+	/**
+	 * @return string
+	 */
+	public function getServiceName()	{
+		return $this->serviceName;
+	}
+
+
+	/**
+	 * @param $serviceName
+	 * @return $this
+	 */
+	public function setServiceName( $serviceName )	{
+		$this->serviceName = $serviceName;
+		return $this;
+	}
+
+
+	/**
+	 * @return string
+	 */
+	public function getServiceTag()	{
+		return $this->serviceTag;
+	}
+
+	/**
+	 * @param $serviceTag
+	 * @return $this
+	 */
+	public function setServiceTag( $serviceTag )	{
```

Comment:
The spacing here looks off. Running the [stylize command](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#mediawiki-php-style-helper) should fix it.